index.rst

```diff
@@ -762,6 +790,10 @@ assembler
 disassembler
     Similar to assembler but for custom deserialization.
 
+subset
+    Optional. If true, indicates that the value for this dataset type should be
+    a list of objects found by calling butler.subset.
```

Comment:
Perhaps clarify that you're calling `butler.subset` with the data ID passed `butler.get(<composite>, ...)`.